internal/provider/vaultsecrets/resource_vault_secrets_app.go

```diff
@@ -184,3 +192,7 @@ func (r *resourceVaultsecretsApp) Delete(ctx context.Context, req resource.Delet
 		return
 	}
 }
+
+func generateResourceName(projectID, appName string) string {
```

Comment:
could we import the resource-manager library here and use the helper func provided there for this generaiton?

Reply:
yeah that's fair!